clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -2829,10 +2829,13 @@ void tools::addOpenMPDeviceRTL(const Driver &D,
     LibraryPaths.emplace_back(LibPath);
 
   OptSpecifier LibomptargetBCPathOpt =
-      Triple.isAMDGCN() ? options::OPT_libomptarget_amdgpu_bc_path_EQ
-                        : options::OPT_libomptarget_nvptx_bc_path_EQ;
+      Triple.isAMDGCN()  ? options::OPT_libomptarget_amdgpu_bc_path_EQ
```

Comment:
Does SPIR-V still rely on the `mlink-builtin-bitcode` hack instead of LTO? The SPIR-V backend should be usable soon from what I hear, so that's the ideal going forward.

Reply:
Yes, we're linking the DeviceRTL using `mlink-builtin-bitcode` today, but we definitely want to use LTO in the future, both for OpenMP and SYCL.

Reply:
Is there a patch for the DeviceRTL build? I've been meaning to gut a lot of that starting with https://github.com/llvm/llvm-project/pull/119091 and soon removing the whole bundling thing so it's more like `libc.a`. ideally we'd have `lib/spirv64-intel-unknown/libomp.a` and the driver just passes `-Xoffload-linker -lomp`.

Reply:
Not yet, we hit some issues in the front end generating OpenMP device code with the SPIR-V target, so we can't build it yet because it doesn't make it through the compiler. Any change should be with fine with us, we should be able to adapt to whatever infrastructure there is to build the DeviceRTL. thanks.